Cargo.toml

```
@@ -11,5 +11,9 @@ repository = "https://github.com/JohnDoneth/hd44780-driver"
 documentation = "https://docs.rs/hd44780-driver"
 readme = "README.md"
 
+[features]
+async = ["embassy-traits"]
+
 [dependencies]
 embedded-hal = "0.2.3"
+embassy-traits = { version = "*", git = "https://github.com/akiles/embassy.git", optional = true }
```

We should pin this to a specific revision since there isn't a stable release as far as I can tell.
```suggestion
embassy-traits = { git = "https://github.com/akiles/embassy.git", rev = "8894684", optional = true }
```

---

The stable release is coming fairly soon (maybe a few months). The reason that embassy-traits is used is that there an official async traits crate like embedded-hal.